haystack/management/commands/update_index.py

```diff
@@ -74,21 +76,24 @@ def worker(bits):
 
 
 def do_update(backend, index, qs, start, end, total, verbosity=1):
+    start_time = now()
     # Get a clone of the QuerySet so that the cache doesn't bloat up
     # in memory. Useful when reindexing large amounts of data.
     small_cache_qs = qs.all()
     current_qs = small_cache_qs[start:end]
 
     index.pre_process_data(current_qs)
 
+    # FIXME: Get the right backend.
+    backend.update(index, current_qs)
+
+    time_delta = now() - start_time
+
     if verbosity >= 2:
         if hasattr(os, 'getppid') and os.getpid() == os.getppid():
-            print("  indexed %s - %d of %d." % (start + 1, end, total))
+            logger.info("  indexed {} - {} of {} in {}.  Took {}".format(start + 1, end, total, time_delta))
```

Comment:
why is it start + 1? Is it because it is zero indexed? If that were the case, would end need to be incremented too?